docs/core/additional-tools/dotnet-svcutil.xmlserializer-guide.md

```diff
@@ -11,64 +17,79 @@ You can use the command `dotnet --info` to check which versions of .NET Core SDK
 
 To use dotnet-svcutil.xmlserializer in a .NET Core console application:
 
-1. Create a WCF Service named 'MyWCFService' using the default template 'WCF Service Application' in .NET Framework.  Add ```[XmlSerializerFormat]``` attribute on the service method like the following
-    ```c#
-    [ServiceContract]
-    public interface IService1
-    {
-        [XmlSerializerFormat]
-        [OperationContract(Action = "http://tempuri.org/IService1/GetData", ReplyAction = "http://tempuri.org/IService1/GetDataResponse")]
-        string GetData(int value);
-    }
-    ```
-2. Create a .NET Core console application as WCF client application that targets at netcoreapp 2.1, e.g. create an app named 'MyWCFClient' with the command,
-    ```
-    dotnet new console --name MyWCFClient
-    ```
-    Make sure your csproj targets a netcoreapp 2.1. This is done using the following XML element in your .csproj file
-    ```xml
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    ```
+1. Create a WCF Service named 'MyWCFService' using the default template 'WCF Service Application' in .NET Framework.  Add `[XmlSerializerFormat]` attribute on the service method like the following:
+
+```csharp
```

Comment:
fenced code should be indented so that it aligns with the numbered list. That's true of all of the fenced code in this topic.